infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -481,6 +469,11 @@ private synchronized void startSnapshotApplyAsync(LogReplication.LogReplicationE
 
     private synchronized void startSnapshotApply(LogReplication.LogReplicationEntryMsg entry) {
         log.debug("Entry Start Snapshot Sync Apply, id={}", entry.getMetadata().getSyncRequestId());
+        //set data_consistent as false and clear streams which has evidenced data from active and the streams
+        // which have been locally written (aimed for replication) but yet were not replicated from active to standby (empty on active)
+        // Note: the locally updated streams must be cleared or we could pollute the state of the DB
+        setDataConsistentWithRetry(false);
+        snapshotWriter.clearLocalStreams();
```

Comment:
given that you changed the semantics this method should be named clearStreams, right? as it now clears local written streams + replicated streams (and not only local like before).

Reply:
agree, changed it to clearStreams().